### PR TITLE
fix for poolswap

### PR DESCRIPTION
**Pull request by thedoublejay**
#### What kind of PR is this?:

/kind fix

#### What this PR does / why we need it:
1. Added disable for Token B input as we don't allow updating it yet
2. Added params on cancel
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx [hunk 1]
```diff
@@ -85,7 +85,8 @@ export function PoolSwapScreen ({ route }: Props): JSX.Element {
         tokenA,
         tokenB,
         swap,
-        fee
+        fee,
+        poolpair
```
Comment (ivan-zynesis) [inline app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx:89]: where is this used?

Comment (thedoublejay) [reply app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx:89]: when cancelling, need to pass params again.

Comment (thedoublejay) [reply app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx:89]: https://github.com/DeFiCh/wallet/blob/141c024490291b2fda3979e39c5bfe4dcd260869/app/screens/AppNavigator/screens/Dex/PoolSwap/ConfirmPoolSwapScreen.tsx#L65

**Comment by fuxingloh** [comment]
image
 

should reset value when we click that button

**Comment by thedoublejay** [comment]
Updated @fuxingloh

**Comment by fuxingloh** [comment]
1 last thing is this should be estimated too:

image

**Comment by thedoublejay** [comment]
@fuxingloh updated title

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**